client/src/pages/index.tsx

```diff
@@ -153,32 +151,35 @@ const IndexPage = () => {
                 </div>
               </div>
             </div>
-
-            {/* aside*/}
-            <aside
-              aria-labelledby="left-sidebar"
-              className="desktop:grid-col-3 usa-prose grid-gap j40-aside"
-              id="left-sidebar">
-              <h2 className={'usa-graphic-list__heading'}>Get Involved</h2>
-              <h3>
+          </section>
+
+          {/* aside */}
+          <aside
+            aria-labelledby="left-sidebar"
+            className={'j40-aside desktop:grid-col-3'}
+            id="left-sidebar">
+            <section
+              className={'usa-prose grid-gap grid-container usa-section'}>
+              <h3 className={'j40-aside-title'}>Get Involved</h3>
+              <h5>
```

Comment:
Based on how [this page](https://designsystem.digital.gov/components/prose/) uses this, I'm thinking the analog is to make `Get Involved` an `<h1>` and then `Send Feedback` an `<h2>`?

These may not match the style spec but they seem like semantically the correct components to use -- if we're using them to get the right font-size it seems we should actually change the spec instead?

Reply:
Yeah, I was visually just trying to match the prototype sizes and style.

Think we need to fix the page font to be `Source Sans Pro` instead of the current `Merriweather` and then maybe the `<h#>` tags will be closer? (An easy way to spot the font difference is by looking at the lower-case `i`).  

All the headers tags on https://designsystem.digital.gov/components/prose/ are using `Merriweather` (as is our page right now), but mocks are using `Source Sans Pro` for the header tags.